components/camel-kafka/src/test/java/org/apache/camel/component/kafka/integration/KafkaConsumerManualCommitIT.java

```diff
@@ -29,9 +29,11 @@
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.api.TestInstance;
 
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 
+@TestInstance(TestInstance.Lifecycle.PER_METHOD)
```

Comment:
AFAIK it is the default, so it should not be needed

Reply:
Hi @essobedo ! Thanks for reviewing! 
It's no longer the default (PER_CLASS inherited from the [parent](https://github.com/apache/camel/pull/7254/commits/b54c747b22baaec4e5faecc5588223d4c380b7d4#diff-51302f36d8ba1b24821b40d3b2b95b37d06a07537e8d965ddfe11c075210239dR36))
Thanks!